tests/unit/postTestBuildComment.ts

```diff
@@ -86,7 +93,7 @@ describe('Post test build comments action tests', () => {
               }
             }
         `);
-        expect(GithubUtils.createComment).toBeCalledTimes(1);
-        expect(GithubUtils.createComment).toBeCalledWith('App', 12, message);
+        expect(createCommentMock).toBeCalledTimes(1);
+        expect(createCommentMock).toBeCalledWith('App', 12, message);
```

Comment:
```suggestion
        expect(createCommentMock).toBeCalledWith('App', '12', message);
```

With this change the test should pass correctly